include/RAJA/policy/cuda/atomic.hpp

```diff
@@ -556,17 +623,15 @@ template <typename T,
           RAJA::util::enable_if_is_none_of<T, cuda_atomicBit_builtin_types>* = nullptr>
 RAJA_INLINE __device__ T cuda_atomicOr(T *acc, T value)
 {
-  return cuda_atomicCAS(acc, [value] (T old) {
+  return cuda_atomicCAS_loop(acc, [value] (T old) {
     return old | value;
   });
 }
 
-template <typename T,
-          RAJA::util::enable_if_is_any_of<T, cuda_atomicBit_builtin_types>* = nullptr>
-RAJA_INLINE __device__ T cuda_atomicOr(T *acc, T value)
-{
-  return ::atomicOr(acc, value);
-}
+/*!
+ * Atomic or via builtin functions was implemented much earlier since atomicLoad
+ * may depend on it.
+ */
```

Comment:
You could forward declare it instead of moving it up, not sure if that will look nicer or not.

Reply:
I think I'd rather just keep a comment here.